parsers/jscript.c

```diff
@@ -1940,8 +1950,7 @@ static bool parseStatement (tokenInfo *const token, bool is_inside_class)
 	bool is_const = false;
 	bool is_terminated = true;
 	bool is_global = false;
-	bool has_methods = false;
-	vString *	fulltag;
+	vString * fulltag;
```

Comment:
Very very minor item. 
Could you this change from this commit?
Doing this in another commit is acceptable, of course.